composer.json

```diff
@@ -39,7 +39,8 @@
     "require-dev": {
         "fzaninotto/faker": "^1.5",
         "graham-campbell/testbench-core": "^1.0",
-        "phpunit/phpunit": "^4.7.6"
+        "phpunit/phpunit": "^4.7.6",
+        "laravel/homestead": "dev-master"
```

Comment:
I'm saying all you need to do is not have it in the composer.json, and everything still works...